docs/tutorial/using-checkbox/basic-running.rst

```diff
@@ -0,0 +1,394 @@
+    │[X]    A job that displays an environment variable, if set                    │
+    │[X]    A job that always fails                                                │
+    │[X]    A job that is skipped because it depends on a job that fails           │
+    │[X]    A job that requires a resource and it is available                     │
+    │[X]    A manual job                                                           │
+    │[X]    A job that requires a resource but it's not available                  │
+    │[X]    A job that always passes                                               │
+    │[X]    A job that depends on other job that passes                            │
+    │[X]    A semi-automated job where the outcome is set automatically            │
+    │[X]    A semi-automated job where the user manually sets the outcome          │
+    │[X]    A job that generates different resources for tutorial purposes         │
+    │                                                                              │
+    └──────────────────────────────────────────────────────────────────────────────┘
+     Press (T) to start Testing                                            (H) Help
+
+Now we have some idea of what is going to be executed. Note that, by default,
+all the jobs in the test plan are selected (you can see the ``[X]`` mark next
+to them). You can toggle a job selection by highlighting it and pressing the
+``space`` key. If you do this while highlighting a category, all the jobs
+in this category will be (de)selected. Finally, try to press the ``d``
+key. What happened? All the jobs have been deselected! Press the ``s``
+key to select them all again.
+
+Press ``t`` to start the test run. A few automated jobs will be executed,
+and their outcome automatically set before Checkbox moves on to the next one:
+
+.. code-block:: none
+
+    ========[ Running job 1 / 11. Estimated time left (at least): 0:03:00 ]=========
+    --------------------------[ A job that always passes ]--------------------------
+    ID: com.canonical.certification::tutorial/passing
+    Category: com.canonical.certification::tutorial
+    ... 8< -------------------------------------------------------------------------
+    This job passes!
+    ------------------------------------------------------------------------- >8 ---
+    Outcome: job passed
+    ========[ Running job 2 / 11. Estimated time left (at least): 0:03:00 ]=========
+    --------------------------[ A job that always fails ]---------------------------
+    ID: com.canonical.certification::tutorial/failing
+    Category: com.canonical.certification::tutorial
+    ... 8< -------------------------------------------------------------------------
+    This job fails!
+    ------------------------------------------------------------------------- >8 ---
+    Outcome: job failed
+    (...)
+
+For each job, we can see:
+
+- How many jobs have run (2), and how many jobs there are in total (11)
```

Comment:
How about "The number of jobs that have run (2) and the total number of jobs scheduled (11)"?

Reply:
Fixed in https://github.com/canonical/checkbox/pull/705/commits/171502bdca37d5a4044eaf85ef5d19ac5526a3b8. Thanks!